src/OpenTelemetry.Api/Internal/Utils.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="Utils.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Internal
+{
+    internal static class Utils
+    {
+        public static bool IsSortedAndDistinct(double[] values)
```

Comment:
Minor: I think this method is less likely to be used by other places, perhaps:
1. move it to the `MeterProviderBuilderExtensions.cs` as a private static method.
2. change the name to indicate the specific intention (e.g. `ValidateHistogramBucketBounds`).